lib/interactiveCLI.js

```diff
@@ -0,0 +1,70 @@
+
+import inquirer from "inquirer"
+import pathModule from "path"
+import chalk from "chalk"
+
+
+import { createComponent, loadOptionsInConfigFileIfExists } from "../lib/core"
+
+const askQuestions = (options) => {
+	const questions = [
+		{
+			choices: ["Component", "Hook"],
+			message: "What would you like to create?",
+			name: "type",
+			type: "list",
+		},
+		{
+			message: "Enter the component's path (components will take their name from the last folder of the path).",
+			name: "path",
+			type: "input",
+			when: (answers) => answers.type === "Component",
```

Comment:
Hook feature is now merged. Would you mind implementing the hook part also ?

Reply:
It should be ok with https://github.com/reacli/cli/pull/43/commits/ec782719db342247d5379b2162c9513342a0fa75.